op-supervisor/supervisor/backend/superevents/events.go

```diff
@@ -119,15 +119,6 @@ func (ev LocalDerivedEvent) String() string {
 	return "local-derived"
 }
 
-type LocalDerivedDoneEvent struct {
-	ChainID eth.ChainID
-	Derived types.DerivedBlockRefPair
-}
-
-func (ev LocalDerivedDoneEvent) String() string {
-	return "local-derive-done"
-}
-
```

Comment:
FYI this event was added specifically to ensure that Unsafe<>Safe consistency checks would happen *after* the write to the database was complete. Otherwise, it would have been possible for the check to happen before the data lands on the DB, affecting the outcome of the consistency check.

Reply:
the event it uses now is also only emitted after the DB write.

Reply:
Ah yep, you are correct. Thanks.